assets/models/system/stabilityai-stable-diffusion-2-1/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: finetuning-image-models
-  container_path: HuggingFace-mlflow/stabilityai-stable-diffusion-2-1/1702314884/mlflow_model_folder
+  container_path: HuggingFace-mlflow/stabilityai-stable-diffusion-2-1/1702902281/mlflow_model_folder
```

Comment:
hoping that this is expected in this PR?

Reply:
Yes, the artifacts are changes for SD models. Taking this change in same pr.